.github/CHANGELOG.md

```diff
@@ -102,6 +102,7 @@
 1. [ECAM] Implemented proper FWC flight phases - @beheh (Benedict Etzel)
 1. [ECAM] Updated ECAM messages - @beheh (Benedict Etzel)
 1. [CDU] New improved RADNAV page | Ability to tune navaids with identifier - @St54Kevin (Kevin Karas)
+1. [General] Added aircraft type reporting for the live map - @nistei (nistei)
```

Comment:
No need for changelog, this whole feature was introduced in 0.5.0

Reply:
Thought so